src/DynamoCoreWpf/Controls/NotificationsControl.xaml

```diff
@@ -0,0 +1,49 @@
+﻿<UserControl x:Class="Dynamo.Wpf.Controls.NotificationsControl"
```

Comment:
The NotificationsControl replaces the "warning_text" object in the DynamoView. This control instantiates a NotificationsViewModel object to use as its DataContext. The NofificationsViewModel listens for NotificationPosted events on the Notifications singleton and store the message and the warning level on two public properties bound to the UI.